aiounifi/interfaces/events.py

```diff
@@ -0,0 +1,124 @@
+"""Manage events from UniFi Network Controller."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any, Callable, Final, Optional
+
+from ..models.event import EventKey, MessageKey, WebsocketData
+
+LOGGER = logging.getLogger(__name__)
+
+
+SubscriptionType = tuple[
+    Callable[[WebsocketData], set[str]],
```

Comment:
This type should include Event:
```suggestion
    Callable[[WebsocketData], set[str | Event]],
```
to match with https://github.com/Kane610/aiounifi/blob/21b9018427155d2c9b339c4e2125f3d2409f1c42/aiounifi/interfaces/api.py#L59